syntaxes/omt.tmLanguage.json

```diff
@@ -90,6 +90,17 @@
                     "contentName": "meta.embedded.interpolated.odt",
                     "patterns": [{ "include": "#_interpolated-odt" }]
                 },
+                {
+                    "match": "(?:\\G|^) +(reason): *([\\w ]*)(.*)$",
```

Comment:
Double check brackets in expression
`(?:\\G|^) +(reason): *(\\w *)(.*)$`

Reply:
The square brackets are absolutely necessary. The way it is now matches for example "Hello World", without the brackets it would match "H         ".